performance/pom.xml

```diff
@@ -118,6 +150,7 @@
                 <dependency>
                     <groupId>org.hibernate</groupId>
                     <artifactId>hibernate-validator</artifactId>
+                    <version>5.1.3.Final</version>
```

Comment:
Don't think so as it's a profile used to run 5.1.x.

That being said, maybe we would need a 5.3 target to compare 6 to 5.3? And maybe a 5.2 target to compare 5.3 to 5.2 as 5.3 is not released yet and we would like to ensure we don't have any performance regression when we release it.